Mathlib/Algebra/TrivSqZeroExt.lean

```diff
@@ -738,6 +738,32 @@ def inlHom [Semiring R] [AddCommMonoid M] [Module R M] [Module Rᵐᵒᵖ M] : R
 
 end Mul
 
+section Inv
+
+variable {R : Type u} {M : Type v}
+
+instance inv [Neg M] [Inv R] [SMul Rᵐᵒᵖ M] [SMul R M] : Inv (tsze R M) :=
+  ⟨fun b => (b.1⁻¹,  - (b.1⁻¹ •> (b.2 <• b.1⁻¹)))⟩
+
+theorem fst_inv [Neg M] [Inv R] [SMul Rᵐᵒᵖ M] [SMul R M]
+    (x : tsze R M) : fst x⁻¹ = (fst x)⁻¹ :=
+      rfl
+
+theorem snd_inv [Neg M] [Inv R] [SMul Rᵐᵒᵖ M] [SMul R M] (x : tsze R M) :
+    snd x⁻¹ = - ((fst x)⁻¹ •> (snd x <• (fst x)⁻¹)) :=
+      rfl
+
+theorem TrivEqZeroExt.mul_inv_cancel [DivisionRing R] [AddCommGroup M]
```

Comment:
What you've written gives this lemma the name `TrivSqZeroExt.TrivEqZeroExt.mul_inv_cancel` (note also the typo). I think what you probably want is
```suggestion
protected theorem mul_inv_cancel [DivisionRing R] [AddCommGroup M]
```
where `protected` avoids this getting confused with the global `mul_inv_cancel`.